src/main/java/org/isf/admission/service/AdmissionIoOperations.java

```diff
@@ -96,9 +96,9 @@
 
 	/**
 	 * Returns all patients based on the applied filters.
-	 * @param admissionRange the patient admission range
-	 * @param dischargeRange the patient discharge range
-	 * @param searchTerms the search terms to use for filter the patient list, <code>null</code> if no filter have to be applied.
+	 * @param admissionRange (two-dimensions array) the patient admission dates range, both {@code null} if no filter have to be applied.
+	 * @param dischargeRange (two-dimensions array) the patient discharge dates range, both {@code null} if no filter have to be applied.
```

Comment:
This is a small improvement; change:

 `* @param admissionRange (two-dimensions array) the patient admission dates range, both {@code null} if no filter have to be applied.`
  `* @param dischargeRange (two-dimensions array) the patient discharge dates range, both {@code null} if no filter have to be applied.`

to 

`* @param admissionRange (two-dimensional array) the patient admission dates range, both {@code null} if no filter is to be applied.`
  `* @param dischargeRange (two-dimensional array) the patient discharge dates range, both {@code null} if no filter is to be applied.`

Reply:
Fixed, thanks!